site/content@beta/backend-apis/about-apis.md

```diff
@@ -0,0 +1,433 @@
+---
+weight: 20
+---
+
+# About our APIs
+
+There are two primary Chat APIs:
+
+- [Agent Chat API](https://developers.livechatinc.com/beta-docs/agent-chat-api/), which serves to join a chat as an agent.
+- [Customer Chat API](https://developers.livechatinc.com/beta-docs/customer-chat-api/), which serves to join a chat as a customer.
+
+The **Agent Chat API** contains a different set of methods than the **Customer Chat API**. We separated both APIs as they cover different use cases. 
+
+## RTM API vs. Web API
+
+Both APIs have much in common. One similarity is that they're both **Real Time Messaging APIs (RTM APIs)**, which translates to enabling real-time communication. Websocket transport allows for **pushes**, which are server-client methods used to keep application state up-to-date. In case of the Agent App and the Chat Widget, the continuous connection is crucial. That's why they both implement RTM API.
+
+|**Characteristics**|     **RTM API**      | **Web API**               |
+|:-----------------:|:--------------------:|:-------------------------:|
+|**connection type**|   websocket          |  independent requests     |
+| **makes use of**  |   pushes             |    webhooks               |
+|**implemented by** |Agent App, Chat Widget|external apps, integrations|
+
+Integrations built upon the LiveChat Platform don't usually need to keep the connection open continuously. In the context of app functioning, short delays are insignificant. It's the reason why **Web API** is more commonly used than **RTM API** when building an integration. Just like REST API, Web API is based on sending the **request** and getting the **response**. Instead of pushes, Web API makes use of **webhooks**.
+
+## Events
```

Comment:
IMO zdecydowanie lepszym rozwiązaniem byłoby przekazanie PATH, SAMPLE REQUEST i SAMPLE RESPONSE dla jednego z eventów, a następnie przekierowanie do Agent Chat API/Customer Chat API po resztę eventów. Chodzi o zrobienie delikatnego wstępu, który jasno wyjaśnia jak użyć API i przedstawia przykład.
Przykład z REST API, który robi to świetnie: https://developers.livechatinc.com/docs/rest-api/#agents-available-paths

Reply:
@bkubiak @fjaskolski co myślicie o pomyśle Orzecha? 

Lepiej, żeby w tym dokumencie był 1 dokładny przykład użycia z wykorzystaniem jednego z eventów, a inne rodzaje eventów tylko wymienione (+ linki)? 

Czy ten dokument powinien z grubsza wyjaśniać jak użyć API czy to powinno tłumaczyć Agent Chat API/ Customer Chat API?

Reply:
Nie jestem pewien, taki przykład powinien być już chyba w opisie danego API, bo jednak wystepują pewne różnice pomiędzy nimi